app/_src/gateway/production/monitoring/healthcheck-probes.md

```diff
@@ -10,6 +10,34 @@ The readiness check endpoint returns a `200 OK` response when {{site.base_gatewa
 {:.note}
 > **Note:**  The readiness endpoint does not return detailed information about the node status.
 
+## Types of Health Checks
+
+For each {{site.base_gateway}} node, there exist two distinct health checks (also known as "probes"):
+
+* Liveness: This `/status` endpoint responds with a `200 OK` status if Kong is running. The request will fail either with a `500 Internal Server Error` or no response if Kong is not running. You can send a GET request to check the liveness of your {{site.base_gateway}} instance:
+  ```sh
+  # Replace `localhost:8100` with the appropriate host and port for
+  # your Status API server
+  
+  curl -i http://localhost:8100/status
+  ```
+
+* Readiness: This `/status/ready` endpoint responds with a `200 OK` status if Kong has successfully loaded a valid configuration and is ready to proxy traffic. The request will fail either with a `500 Internal Server Error` or no response if Kong is not ready to proxy traffic yet. You can send a GET request to check the readiness of your {{site.base_gateway}} instance:
+  ```sh
+  # Replace `localhost:8100` with the appropriate host and port for
+  # your Status API server
+  
+  curl -i http://localhost:8100/status/ready
+  ```
+
+These two types of health checks for {{site.base_gateway}} are modeled on how [Kubernetes defines](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) health check probes.
+
+It is strongly recommended that a component (i.e. load balancer) perform the readiness health check before sending traffic. This ensures that a {{site.base_gateway}} node has not only successfully started up but has also finished loading up the configuration and is ready to receive proxy traffic. 
+
+The liveness health check may return a 200 OK before the readiness health check does while it's running but still loading the full configuration. If a component only monitors the liveness probe to decide when to send traffic to {{site.base_gateway}}, then there will be a short period of time where requests will be met with a `404 Not Found` response the {{site.base_gateway}} is ready to proxy traffic. We recommend using the readiness probe over the liveness probe especially in production environments.
```

Comment:
```suggestion
The liveness health check may return a 200 OK response before the readiness health check does. 
Even if {{site.base_gateway}} is running, it may still be loading the full configuration, which means it is live but not ready.
If a component only monitors the liveness probe to decide when to send traffic to {{site.base_gateway}}, there will be a short period of time where requests will be met with a `404 Not Found` response before the {{site.base_gateway}} is ready to proxy traffic. 
We recommend using the readiness probe over the liveness probe, especially in production environments.
```
Some rephrasing for clarity.